extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -1590,6 +1602,20 @@ namespace Microsoft.Windows.DevHome.SDK
         Windows.Foundation.IAsyncOperationWithProgress<QuickStartProjectResult, QuickStartProjectProgress> GenerateAsync();
     }
 
+    // Extensions implement this and is provided to Dev Home when CreateChatStyleGenerationOperation is called.
+    // This allows to have an async operation can be started but also additional context reported back to Dev Home
+    // as progress is being made such as a detailed log out output through the adaptive card session.
+    [experimental]
+    [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 8)]
+    interface IQuickStartChatStyleGenerationOperation
+    {
+            // Used to create a response based on whether the user's prompt is a valid prompt.
+            // A valid prompt is determined by the denotion of a project type, having a clear objective,
```

Comment:
The SDK should not define what a valid prompt is.  Implementers will do that.

Reply:
Got it, that makes sense. I removed the rest of the sentences about defining what a valid prompt is.